Tip to install email-validator package to use email validation (#3367)
fastapi/fastapi: docs/en/docs/tutorial/response-model.md

## Return the same input data

> !!! info
>     To use email validator (`EmailStr` type), first install <a href="https://github.com/JoshData/python-email-validator" class="external-link" target="_blank">`email-validator`</a> package.
> 
>     E.g. `pip install email-validator`.
> 

Reviewer: Personally I rather see this as a note under the code worded more like:
```suggestion
!!! note
    `EmailStr` uses <a href="https://github.com/JoshData/python-email-validator" class="external-link" target="_blank">`email-validator`</a>, you will have to install this package to use `EmailStr`.

```
